pages/portfolio.html

```diff
@@ -1,12 +1,95 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Anika's Portfolio: Projects</title>
+    <link href="/styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+    <header>
+        <h1>Anika's Personal Portfolio: Projects</h1>
+    <nav>
```

Comment:
Looks like nav is a child to the header tag, but the indentation is one level off. Think it needs to be indented one more time